examples/bert/bert_ptq_qnn_ep.json

```diff
@@ -0,0 +1,85 @@
+{
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "hf_config": {
+                "model_name": "Intel/bert-base-uncased-mrpc",
+                "task": "text-classification",
+                "dataset": {
+                    "data_name":"glue",
+                    "subset": "mrpc",
+                    "split": "validation",
+                    "input_cols": ["sentence1", "sentence2"],
+                    "label_cols": ["label"],
+                    "batch_size": 1
+                }
+            }
+        }
+    },
+    "evaluators": {
```

Comment:
Same for the evaluations. I am currently working on updating the support for qnn ep evaluations. 

I think it might be better to just remove this example for now and add it when the e2e workflows has been validated for qnn ep. Especially, since this example is not related to the PR.